bazel/opentelemetry_proto.BUILD

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+proto_library(
```

Comment:
How come we have to supply a build_file override for otel protos instead of upstream doing the right thing?

Reply:
That would be better, for sure.

But there's no protobuf support in https://github.com/open-telemetry/opentelemetry-proto and I didn't want to hold up development waiting for it. I'm fine removing when it gets added.

Reply:
That's fair. Let's upstream it later.